docs/community/chat.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Chat
+legacyUrl: https://docs.ipfs.io/community/irc/
+description: How to connect with the IPFS community using synchronous chat.
+---
+
+# Matrix Chat
+
+If IRC is your go-to for discussions and help on dev topics, there are several IRC channels on Freenode dedicated to IPFS, and it’s a great place for live chat:
+
+- Use [`#lobby:ipfs.io`](https://app.element.io/#/room/#lobby:ipfs.io) for general IPFS usage and community chat.
```

Comment:
In effort to decrease noise and funnel users to specific topic areas, we may want to add rooms for libp2p, ipld, multiformats ones to this list.

We also need to have a link for people interested discussing Filecoin (I believe there is slack, weChat and Matrix?), so less people will  join IPFS lobby asking questions about FIL.

Reply:
I wonder if it'd be worth just merging libp2p, ipld, and multiformats into #ipfs as I don't typically see a lot of activity in the other rooms. Mostly because while the topic'd rooms are more specific, and ultimately get specialised answers, I worry users might leave before they get their answer if there's little-to-no activity normally.

Definitely agree on linking to the FIL community somewhere. The FIL Slack also recently got IPFS related channels too (which are the ones bridged to the Discord/Matrix rooms).

Reply:
Actually scratch that, the rooms #ipld and #libp2p are already live on the super bridge, might as well point people to them.